typescript/cli/examples/ccip/multi-chain-ccip-warp-route.yaml

```diff
@@ -0,0 +1,81 @@
+arbitrum:
+  mailbox: '0x979Ca5202784112f4738403dBec5D0F3B9daabB9'
+  owner: '0x3f13C1351AC66ca0f4827c607a94c93c82AD0913'
+  type: synthetic
+  interchainSecurityModule:
+    type: domainRoutingIsm
+    owner: '0x3f13C1351AC66ca0f4827c607a94c93c82AD0913'
+    domains:
+      polygon:
+        type: ccipIsm
+        originChain: polygon
+      base:
+        type: ccipIsm
+        originChain: base
+  hook:
+    type: aggregationHook
+    hooks:
+      - '0x3b6044acd6767f017e99318AA6Ef93b7B06A5a22'
```

Comment:
whats this?

Reply:
ah i think this is a leftover from before the default hook config - will replace with that